php/libraries/NDB_Page.class.inc

```diff
@@ -736,6 +734,15 @@ class NDB_Page
     {
     }
 
+    /**
+     * Provides setup for a page that is done after _setup, when $this->form
+     * et al has been initialized.
+     *
+     * @return none
+     */
+    function commonPageSetup()
```

Comment:
not required anymore I think.

Reply:
Thanks, removed it. We'll see if Travis says I missed anywhere..